documentation/general/deployment.md

```diff
@@ -35,7 +35,7 @@ traffic to the new task away from the old task. This is called _draining_. Once
 the old task is drained, it is spun down and the only task left running is the
 new one. The same process is followed when there are multiple tasks and each
 task is drained one at a time following our
-[zero-downtime approach to deployments](https://docs.openverse.org/guides/zero-downtime-database-management.html).
+[zero-downtime approach to deployments](https://docs.openverse.org/general/zero_downtime_database_management.html).
```

Comment:
Also the same for the remaining `docs.openverse.org` URLs.

Reply:
@dhruvkb Ah, I did the other one in f3be2ea3a82c1c95cdedea448311c51f0144a0f8. One of the references is in a PR template so that one will need to stay a link to `docs.openverse.org` I think.